experimental/uefi/runtime/src/remote_attestation.rs

```diff
@@ -22,12 +22,31 @@
 extern crate alloc;
 
 use alloc::vec::Vec;
-use anyhow::Context;
-use oak_remote_attestation_sessions::{SessionId, SessionState, SessionTracker};
+use anyhow::{bail, Context};
+use oak_remote_attestation_sessions::{SessionId, SessionState, SessionTracker, SESSION_ID_LENGTH};
 
 /// Number of sessions that will be kept in memory.
 const SESSIONS_CACHE_SIZE: usize = 10000;
 
+fn deserialize_request(msg: Vec<u8>) -> anyhow::Result<(SessionId, Vec<u8>)> {
+    let mut session_id: SessionId = [0; SESSION_ID_LENGTH];
+    let mut request_body: Vec<u8> = Vec::with_capacity(msg.len() - SESSION_ID_LENGTH);
+
+    if msg.len() < SESSION_ID_LENGTH {
+        bail!("Message too short to contain a SessionId. The length of a SessionId is {} bytes, the message received contained only {} bytes", SESSION_ID_LENGTH, msg.len());
```

Comment:
nit: this is a very long statement, and the formatting doesn't formact insidde macros. Perhaps break the string manually into multiple lines.